### DISCUSSION TITLE
Dashboard Grid: Show time highlight on all visible charts

### DISCUSSION THREAD

**Discussion by natehitze-eventlink** [uptrace/uptrace#327] [Ideas]
![image](https://github.com/uptrace/uptrace/assets/151092365/7d5ae665-7b27-4edc-8c33-a74a06e708e7)

Currently this only shows on the single chart you're hovering mouse over. It would be really helpful if it was on all charts in view. That allows you to easily reference the same point in time on multiple charts.

**Comment by vmihailenco** [comment]
This is an easy one and I've just added it to the Cloud: https://app.uptrace.dev/metrics/1/2/grid . Eventually it will be backported to the community edition as well.

**Reply by natehitze-eventlink** [reply]
Thanks! I use Cloud and found a minor bug with this. I have a dashboard with 3 charts and one of them doesn't receive the lines when hovering over either of the 2 charts on the left:
![image](https://github.com/uptrace/uptrace/assets/151092365/82840e61-48a1-41db-a31d-376328305afa)

If I mouse over the chart on the right, it _does_ put the line in all 3 charts.

**Reply by vmihailenco** [reply]
@natehitze-eventlink could you email me (support@uptrace.dev) a link to the dashboard with the permission to check it out? I can't seem to reproduce it locally...
